src/client/types.ts

```diff
@@ -27,7 +26,8 @@ export interface Logger {
 }
 
 export interface ClientSideInstrumentation {
-  onApiCall(stackTrace: ParsedStackTrace): ((error?: Error) => void) | undefined;
+  onApiCallBegin(apiCall: string): { userObject: any };
```

Comment:
I think we can replace `{ userObject: any }` in both methods with `any` and it should be equivalent?

Reply:
the instrumentation end uses the fact that it is an envelope...

Reply:
And more importantly, this documents it, right?